packages/frontend/src/stores/messagelist.ts

```diff
@@ -53,9 +53,9 @@ const defaultState = () =>
   }) as MessageListState
 
 /*
- * A hook to read a portion of messages(a view) for a given chat. It creates a store(MessageListStore)
+ * A hook to read a portion of messages(a view) for a given chat. It uses the passed store(MessageListStore)
  * for the given chat and account and loads messages on it. It always has a maximum specified number
- * of messages as per PAGE_SIZE constant.
+ * of messages as per PAGE_SIZE constant. The store is also used in useDraft to avoid redundant data loading.
```

Comment:
With https://github.com/deltachat/deltachat-desktop/pull/5471/files#r2441759745 having been addressed, the comments also needs to get reverted. Also